src/settingsdialog.cpp

```diff
@@ -26,9 +27,9 @@
 #include "hotspot-config.h"
 
 namespace {
-KConfigGroup config()
+auto config()
```

Comment:
use KSharedConfig::Ptr here please instead of auto, I don't think we rely on c++17 yet? or is this valid c++14 already?

Reply:
seems to be valid, but I changed it anyway